tests/functional_tests/cold_signing.py

```diff
@@ -160,6 +166,25 @@ def transfer(self):
         res = self.cold_wallet.get_tx_key(txid)
         assert len(res.tx_key) == 64
 
+        self.hot_wallet.rescan_blockchain(hard = True)
+
+        res = self.hot_wallet.export_outputs(all_ = True)
+        self.cold_wallet.import_outputs(res.outputs_data_hex)
+        res = self.cold_wallet.export_key_images(all_ = True)
+        self.hot_wallet.import_key_images(res.signed_key_images, offset = res.offset)
+
+        res = self.hot_wallet.get_transfers()
+        assert len([x for x in (res['in'] if 'in' in res else []) if x.txid == txid]) == 0
```

Comment:
Of note, this is the only assertion that would fail without the change to wallet2